client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -244,6 +244,13 @@ const CheckoutThankYou = React.createClass( {
 			/* eslint-enable wpcalypso/jsx-classname-namespace */
 		}
 
+		// Rebrand Cities thanks page
+		if ( isRebrandCitiesSiteUrl( this.props.selectedSite.URL ) && PLAN_BUSINESS === this.props.selectedSite.plan.product_slug ) {
```

Comment:
ah yeah - that's the wrong site property :/ i'll pull your branch into mine and fix that